server/controllers/temporary-accommodation/manage/assessmentsController.ts

```diff
@@ -112,4 +114,18 @@ export default class AssessmentsController {
       })
     }
   }
+
+  update(): RequestHandler {
+    return async (req: Request, res: Response) => {
+      const callConfig = extractCallConfig(req)
+
+      const assessment = await this.assessmentsService.findAssessment(callConfig, req.params.id)
+
+      assessment.status = req.params.status as Assessment['status']
+      await this.assessmentsService.updateAssessment(callConfig, assessment)
+
+      req.flash('info', `Assessment updated status updated to "${lowerCase(assessment.status)}"`)
```

Comment:
I'm a bit wary about tying user-facing naming to API naming - thought on `lowerCase(statusName(assessment.status))`?

Reply:
yeah makes sense